### PR TITLE
Add new section to 389 Directory Server chapter

### PR DESCRIPTION
**Pull request by cjschroder**
Add new section 'Synchronizing with Microsoft Active Directory' to 389 Directory Server chapter

### PR creator: Are there any relevant issues/feature requests?

* jsc#DOCTEAM-455

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/security_ldap_active_directory_sync.xml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  <para>
+    You must first chooose the direction of your data flow. There are three 
+    options: from &ada; to &ds389a;, from &ds389a; to &ada;, or bi-
+    directional.
+  </para>
+  <note>
+    <title>No password synchronization</title>
+    <para>
+      Passwords cannot be synchronised between &ds389a; and &ad;.
+      This may change in the future, to support &ad; -> &ds389a; password
```
Comment (Firstyear) [inline xml/security_ldap_active_directory_sync.xml:40]: probably should be "ad to ds" not an arrow :)

File: xml/security_ldap_active_directory_sync.xml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      flow.
+    </para>
+    </note>
+    <para>
+      Your topology will look like the following diagram. The &ds389; and 
+      &ad; topologies may differ, but the most important factor is to have 
+      only a single connection between &ds389a; and &ad; exists. It is very
```
Comment (Firstyear) [inline xml/security_ldap_active_directory_sync.xml:47]: to have only a single connection between ds and ad exists. Seems awkwardly worded, probably my fault.

File: xml/security_ldap_active_directory_sync.xml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      environment, so you should use strong, randomly-generated service 
+      account passwords for these accounts, and carefully audit membership to this group.
+    </para>
+  </warning>
+  <para>
+    You should also create a service account that is a member of this group. 
+  </para>
+  <para>
+    Your &ad; environment must have certificates configured for LDAPS to 
+    ensure that authentication between &ds389a; and &ada; is secure. The
```
Comment (Firstyear) [inline xml/security_ldap_active_directory_sync.xml:94]: "Authentication with GSSAPI ..." not "The gssap ..."

**Review by Firstyear — CHANGES_REQUESTED** [review]

File: xml/security_ldap_active_directory_sync.xml (hunk 2 above)
Comment (cjschroder) [reply xml/security_ldap_active_directory_sync.xml:47]: :+1:

File: xml/security_ldap_active_directory_sync.xml (hunk 1 above)
Comment (cjschroder) [reply xml/security_ldap_active_directory_sync.xml:40]: :+1:

File: xml/security_ldap_active_directory_sync.xml (hunk 3 above)
Comment (cjschroder) [reply xml/security_ldap_active_directory_sync.xml:94]: :+1:

**Comment by cjschroder** [comment]
SLE15SP3 78cb33ba8
SLE15SP2 47d6a8c21

**Pull request merged**